connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -166,12 +167,20 @@ object KustoWriter {
         if (writeOptions.isTransactionalMode) {
           finalizeIngestionWhenWorkersSucceeded(
           tableCoordinates, batchIdIfExists, tmpTableName, partitionsResults, writeOptions,
-          crp, tableExists, rdd.sparkContext, authentication, kustoClient)
+          crp, tableExists, rdd.sparkContext, authentication, kustoClient, sinkStartTime)
         }
       }
     }
   }
 
+  def getCreationTime(ingestionProperties: SparkIngestionProperties, tableCoordinates: KustoCoordinates): Instant = {
+    val creationTime = Option(ingestionProperties.toIngestionProperties(tableCoordinates.database, tableCoordinates.table.get).getAdditionalProperties.get("creationTime"))
+    creationTime match {
+      case Some(creationTimeVal) => Instant.parse(creationTimeVal)
```

Comment:
Do we validate this date time format upfront ? If we already have it in the documentation its okay. We have to explicitly call out and even better if we validate this upfront when we parse the options.
If not now, we can add a TODO that we need to add this at some point